padertorch/contrib/examples/vae_mnist/mnist_vae_example.py

```diff
@@ -145,13 +147,18 @@ def forward(self, input):
 @ex.config
 def config():
     epochs = 2          # Use resume=True to train for more epochs
-    batch_size = 128
+    batch_size = 10
     resume = False      # PT: Continue from checkpoints
+
+    assert os.environ.get('STORAGE_ROOT') is not None, 'STORAGE_ROOT is not defined in the environment variables'
     logfilepath = os.environ['STORAGE_ROOT'] + '/mnist'  # PT creates log files
```

Comment:
I would choose a storage_dir name which at least gives some indication of the performed task